freshmaker/utils.py

```diff
@@ -30,6 +30,9 @@
 import koji
 import kobo.rpmlib
 
+import backoff
```

Comment:
Optional: being very annoying that's the best practices about order when importing:
``Imports should be grouped in the following order:
Standard library imports.
Related third party imports.
Local application/library specific imports.``

So perhaps I'd put these first.

Reply:
That's not being annoying, don't worry! 😂 
I like to follow good practices 😁 

I ran isort in `utils` and reorganized all the imports. I saw that there were a few others in the wrong place. 
I also ran `isort --check` for the whole project, and seems like pretty much every file could use an import reorg 😂 
We could do that later in a separate commit and put it in the 'git-blame-ignore-revs' file.

Reply:
Oh no :'( great job for checking that! Let's totally fix that. I'll file a card.